### ISSUE TITLE
Doesn't throw error if get_cardinality is called with wrong nodes in  MarkovModel.

### ISSUE THREAD

**Issue by lohani2280** [pgmpy/pgmpy#905]
I think one thing is lacking in this implementation . If node passed while calling get_cardinality, is not present in the model, then it should throw an error. Right now this is not happening.

###  My environment
* version of pgmpy = "0.1.6"
* Operating System = "ubuntu 16.04"

### Steps to reproduce
import numpy as np
from pgmpy.models import MarkovModel
from pgmpy.factors.discrete import DiscreteFactor

student = MarkovModel([('Alice', 'Bob'), ('Bob', 'Charles')])
phi1 = DiscreteFactor(['Alice', 'Bob'], cardinality=[2, 3], values=np.random.rand(6))
phi2 = DiscreteFactor(['Bob', 'Charles'], cardinality=[3, 3], values=np.random.rand(9))
student.add_factors(phi1, phi2)
student.get_cardinality('Alice')
student.get_cardinality('Bob')
student.get_cardinality('Dexter')

### Expected behaviour
>>> 2
>>> 3
>>> ValueError: Node not present in the Graph

### Actual behaviour
>>> 2
>>> 3
>>> None

**Comment by ankurankan** [comment]
@lohani2280 Duplicate #904 ??

**Comment by lohani2280** [comment]
@ankurankan Yeah this is duplicate. Sorry , my bad. Shall i close this?

**Comment by ankurankan** [comment]
@lohani2280 Yes please. I would suggest to close the other one and add any extra information here.
